src/middlewared/middlewared/plugins/cron.py

```diff
@@ -0,0 +1,175 @@
+from middlewared.schema import accepts, Bool, Dict, Int, List, Patch, Str
+from middlewared.service import CRUDService, ValidationErrors
+from middlewared.validators import Range
+
+
+class CronJobService(CRUDService):
+
+    class Config:
+        datastore = 'tasks.cronjob'
+        datastore_extend = 'cron.job.cron_extend'
+        datastore_prefix = 'cron_'
+        namespace = 'cron.job'
```

Comment:
Lets just do "cronjob". I cant think of anything else we would put under "cron" namespace.